tests/e2e/default/targets/target-aggregates.wdio-spec.js

```diff
@@ -0,0 +1,420 @@
+      await utils.revertSettings(true);
+      await browser.deleteCookies();
+      await browser.refresh();
+    });
+
+    afterEach(async () => await utils.revertDb([/^form:/], true));
+
+    it('should display an empty list when there are no aggregates', async () => {
+      await commonPage.goToAnalytics();
+      await targetAggregatesPage.expectModulesToBeAvailable([
+        '#/analytics/targets',
+        '#/analytics/target-aggregates'
+      ]);
+
+      await targetAggregatesPage.goToTargetAggregates(true);
+      expect(await (await targetAggregatesPage.aggregateList()).length).to.equal(0);
+      expect(await (await targetAggregatesPage.loadingStatus()).isDisplayed()).to.be.true;
+      expect(
+        await (await analyticsPage.emptySelectionNoError()).isDisplayed()
+      ).to.be.true;
+    });
+
+    it('should display an error when there are aggregates but no home place', async () => {
+      const settings = await utils.getSettings();
+      const tasks = settings.tasks;
+      tasks.targets.items[0].aggregate = true;
+      await utils.updateSettings({ tasks }, true);
+      await commonPage.closeReloadModal();
+
+      await commonPage.goToAnalytics();
+      await targetAggregatesPage.goToTargetAggregates(true);
+      expect((await targetAggregatesPage.aggregateList()).length).to.equal(0);
+      expect(
+        await (await analyticsPage.emptySelectionError()).isDisplayed()
+      ).to.be.true;
+    });
+  });
+
+  describe('as a user with a home place', () => {
+    const parentPlace = placeFactory.place().build({ type: 'district_hospital' });
+    const otherParentPlace = placeFactory.place().build({ type: 'district_hospital' });
+    const user = userFactory.build({ place: parentPlace._id, roles: ['program_officer'] });
+    const names = ['Clarissa', 'Prometheus', 'Alabama', 'Jasmine', 'Danielle'];
+    const otherNames = ['Viviana', 'Ximena', 'Esteban', 'Luis', 'Marta'];
+    const generatePlace = (parent, idx, otherPlace) => {
+      const place = placeFactory.place().build({ type: 'health_center', parent: { _id: parent._id } });
+      const contact = personFactory.build({
+        name: otherPlace === true ? otherNames[idx] : names[idx],
+        parent: { _id: place._id, parent: place.parent }
+      });
+      place.contact = { _id: contact._id, parent: contact.parent };
+      return [place, contact];
+    };
+    const docs = _.flattenDeep([
+      Array.from({ length: 5 }).map((e, i) => generatePlace(parentPlace, i, false)),
+      Array.from({ length: 5 }).map((e, i) => generatePlace(otherParentPlace, i, true)),
+    ]);
+    const generateTitle = (title) => ({ en: title });
+    const docTags = [
+      // current targets
+      moment().format('YYYY-MM'),
+      // next month targets, in case the reporting period switches mid-test
+      moment().date(10).add(1, 'month').format('YYYY-MM'),
+    ];
+
+    before(async () => {
+      const allDocs = [...docs, ...[parentPlace], ...[otherParentPlace]];
```

Comment:
This is the same but cleaner
```suggestion
      const allDocs = [ ...docs, parentPlace, otherParentPlace ];
```